simple_history/models.py

```diff
@@ -204,14 +208,12 @@ def get_extra_fields(self, model, fields):
 
         user_model = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
 
-        @models.permalink
         def revert_url(self):
```

Comment:
Are there any tests in the test suite that are testing this function? Couldn't find any. Want to add one and include it with the PR?

Reply:
@rossmechanic I added a simple test of the `reverse_url()` output; there was an existing test that called it, but this looks like the first one to sanity check the output.